docs/libcurl/opts/CURLOPT_SERVICE_NAME.3

```diff
@@ -32,15 +32,15 @@ CURLcode curl_easy_setopt(CURL *handle, CURLOPT_SERVICE_NAME, char *name);
 .SH DESCRIPTION
 Pass a char * as parameter to a string holding the \fIname\fP of the service
 for DIGEST-MD5, SPNEGO and Kerberos 5 authentication mechanisms. The default
-service names are "ftp", "HTTP", "imap", "ldap", "pop" and "smtp". This option
-allows you to change them.
+service names are "ftp", "HTTP", "imap", "ldap", "pop", "sieve" and "smtp".
+This option allows you to change them.
 
 The application does not have to keep the string around after setting this
 option.
 .SH DEFAULT
 See above
 .SH PROTOCOLS
-HTTP, FTP, IMAP, LDAP, POP3 and SMTP
+HTTP, FTP, IMAP, LDAP, ManageSieve, POP and SMTP
```

Comment:
Shouldn't this still say POP3?

Reply:
Yes it should. Probably an editing glitch.
Thanks for pointing this out.